src/frontend/src/components/ConfirmDialog/index.tsx

```diff
@@ -14,41 +21,67 @@ export interface ConfirmDialogProps {
   content: string | JSX.Element;
   footer?: string;
   customConfirmButton?: JSX.Element;
+  withCloseIcon?: boolean;
+  disableBackdropClick?: boolean;
+  isConfirmButtonClickable?: boolean;
 }
 
 export default function ConfirmDialog({
   open,
   onClose,
   onConfirm,
   customConfirmButton,
+  disableBackdropClick = true,
   title,
   content,
   footer,
+  withCloseIcon = false,
+  isConfirmButtonClickable = true,
 }: ConfirmDialogProps): JSX.Element {
   const confirmButton = customConfirmButton ?? (
     <Button color="primary" variant="contained" isRounded>
       Continue
     </Button>
   );
 
+  const confirmButtonwithDiv = isConfirmButtonClickable ? (
```

Comment:
If it's not clickable, it feels like there should be some styling change. This has it continue to present visually the same whether clickable or not, right? If that's what we'd want, I think I'd just dynamically change from `onConfirm` to `noop` depending on if it's clickable. But I think the `confirmButton` definition above should have the default case get some sort of styling for the `!isConfirmButtonClickable` case

Reply:
i'm a little confused on what styling you think should be happening if it's not clickable, would you mind elaborating a bit more?

Reply:
the isConfirmButtonClickable prop name is confusing.. I can probably just change the delete tree modal instead to include onConfirm in the button itself instead of passing to the div

Reply:
removed this